namespaces/openstack/openstack.yml

```diff
@@ -0,0 +1,45 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+openstack:
```

Comment:
Remove as it is not really to be used for describing logs coming from the OpenStack fluentd collector/normalizer.